### PR TITLE
set up cypress

### PR DESCRIPTION
**Pull request by mathis-marcotte**
closes https://github.com/StatCan/jupyter-apis/issues/206

### CODE REVIEW

File: frontend/jupyter/cypress/fixtures/pvcs.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "pvcs":[],
+  "success":true,
+  "status":200,
+  "user": null
+}
```
Comment (wg102) [inline frontend/jupyter/cypress/fixtures/pvcs.json:6]: Add new line at end of file

File: frontend/jupyter/cypress/fixtures/settings.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "namespace": "kubeflow-user",
+  "postHeaders": {
+    "X-XSRF-Token": "asdf",
```
Comment (wg102) [inline frontend/jupyter/cypress/fixtures/settings.json:4]: Hardcoded token? 
I might not be understanding the purpose of this file exactly

**Review by wg102 — CHANGES_REQUESTED** [review]
I will be testing the actual instructions as well. Let me know the answers to the comments

File: frontend/jupyter/cypress/fixtures/settings.json (hunk 2 above)
Comment (mathis-marcotte) [reply frontend/jupyter/cypress/fixtures/settings.json:4]: Honestly, this was copy pasted from upstream. And looking at it, the postHeaders isnt even being used so i can just remove it.

File: frontend/jupyter/cypress/fixtures/pvcs.json (hunk 1 above)
Comment (mathis-marcotte) [reply frontend/jupyter/cypress/fixtures/pvcs.json:6]: Why exactly? I'm not doubting you, I just want to understand.

Comment (wg102) [reply frontend/jupyter/cypress/fixtures/pvcs.json:6]: It's a convention? It's also the only file github is complaining about for it.

Comment (mathis-marcotte) [reply frontend/jupyter/cypress/fixtures/pvcs.json:6]: Ok thanks, you are correct that github is complaining for this file and a couple of the other new ones. I'll go do that.

**Review by wg102 — APPROVED** [review]
Passes and results as expected

**Pull request merged**